src/components/RequestRepository/index.tsx

```diff
@@ -164,6 +164,7 @@ const RequestRepository = () => {
                 <th
                   {...column.getHeaderProps(column.getSortByToggleProps())}
                   aria-sort={getColumnSortStatus(column)}
+                  style={{ whiteSpace: 'nowrap' }}
```

Comment:
Should we make a class for this? I didn't see a util class in USWDS. I also think this is one-off styling, so I'm torn here, but open to suggestion.

Reply:
This is fine. I'm not super picky about inline styles.